js/testapps/rag/src/index.ts

```diff
@@ -82,8 +84,23 @@ export default configureGenkit({
     ]),
     chroma([
       {
-        collectionName: 'dogfacts_collection',
+        collectionName: 'my_collection',
         embedder: textEmbeddingGecko,
+        createCollectionIfMissing: true,
+        clientParams: async () => {
+          const host = 'https://chroma-974125110527.us-central1.run.app';
+          const authClient = await auth.getIdTokenClient(host);
+
+          const idToken = await authClient.idTokenProvider.fetchIdToken(host);
+          return {
+            path: `${host}:443`,
```

Comment:
is the `:443` necessary? it's the `https` default port....

Reply:
Yeah, looks like it's not. Removed it